mulighetsrommet-api/src/main/kotlin/no/nav/mulighetsrommet/api/repositories/TiltaksgjennomforingRepository.kt

```diff
@@ -540,4 +540,62 @@ class TiltaksgjennomforingRepository(private val db: Database) {
             .asList
             .let { db.run(it) }
     }
+
+    fun getAllByOrgnr(orgnr: String): QueryResult<List<TiltaksgjennomforingDto>> = query {
+        @Language("PostgreSQL")
+        val query = """
+            select tg.id::uuid,
+                   tg.navn,
+                   tg.tiltakstype_id,
+                   tg.tiltaksnummer,
+                   tg.virksomhetsnummer,
+                   tg.start_dato,
+                   tg.slutt_dato,
+                   t.tiltakskode,
+                   t.navn as tiltakstype_navn,
+                   tg.arena_ansvarlig_enhet,
+                   tg.avslutningsstatus,
+                   tg.tilgjengelighet,
+                   tg.antall_plasser,
+                   tg.avtale_id,
+                   array_agg(a.navident) as ansvarlige,
+                   array_agg(e.enhetsnummer) as navEnheter
+            from tiltaksgjennomforing tg
+                     inner join tiltakstype t on t.id = tg.tiltakstype_id
+                     left join tiltaksgjennomforing_ansvarlig a on a.tiltaksgjennomforing_id = tg.id
+                     left join tiltaksgjennomforing_nav_enhet e on e.tiltaksgjennomforing_id = tg.id
```

Comment:
Trenger kanskje ikke være med hvis det ikke brukes?

Reply:
Godt poeng, ser jeg kan rydde en del ja

Reply:
Alternativt kan vi kanskje legge til `organisasjonsnummer` som input til `getAll` og kalle på den funkasjonen fra service eller externalRoutes (i tillegg til å mappe om fra TiltaksgjennomforingAdminDto til TiltaksgjennomforingDto der)?

Reply:
Ja, det går det også. For å slippe å vedlikeholde en ekstra sql-spørring her tenker du?